### ISSUE TITLE
sfctl throwing azure.servicefabric.models.fabric_error_py3.FabricErrorException: (NotFound) Null

### ISSUE THREAD

**Issue by athros** [microsoft/service-fabric-cli#199]
When reporting a new issue include the following information:

- [ ] `sfctl` version: ```Name: sfctl
Version: 7.1.0
Summary: Azure Service Fabric command line
Home-page: https://github.com/Azure/service-fabric-cli
Author: Microsoft Corporation
Author-email: sfpythoncli@microsoft.com
License: MIT
Location: /Users/athros/azure-work/lib/python3.7/site-packages
Requires: msrestazure, adal, psutil, requests, applicationinsights, portalocker, azure-servicefabric, knack, sfmergeutility, future, msrest```
- [ ] Service Fabric cluster runtime version is `6.4.661.1`
- [ ] Service Fabric hosting location - `Azure`
- ```Error: (NotFound) Null
Traceback (most recent call last):
  File "/Users/athros/azure-work/lib/python3.7/site-packages/knack/cli.py", line 206, in invoke
    cmd_result = self.invocation.execute(args)
  File "/Users/athros/azure-work/lib/python3.7/site-packages/sfctl/entry.py", line 81, in execute
    return super(SFInvoker, self).execute(args)
  File "/Users/athros/azure-work/lib/python3.7/site-packages/knack/invocation.py", line 208, in execute
    cmd_result = parsed_args.func(params)
  File "/Users/athros/azure-work/lib/python3.7/site-packages/knack/commands.py", line 139, in __call__
    return self.handler(*args, **kwargs)
  File "/Users/athros/azure-work/lib/python3.7/site-packages/knack/commands.py", line 246, in _command_handler
    result = op(client, **command_args) if client else op(**command_args)
  File "/Users/athros/azure-work/lib/python3.7/site-packages/azure/servicefabric/service_fabric_client_ap_is.py", line 10553, in get_image_store_root_content
    raise models.FabricErrorException(self._deserialize, response)
azure.servicefabric.models.fabric_error_py3.FabricErrorException: (NotFound) Null```

After running a pip update, it looks like the sfctl package is no longer working with specific commands. `sfctl cluster select` seems to work, `sfctl store root-info` does not along with `cluster health` and more. 

This is after a fresh virtual environment creation this morning. Please let me know how I can help.
